packages/decoder/lib/types.ts

```diff
@@ -171,6 +171,7 @@ export interface EventOptions {
    * the event -- should be returned.  Defaults to `"off"`.
    */
   extras?: ExtrasAllowed;
+  disableChecks?: boolean;
```

Comment:
The option needs a documentation comment, we use typedoc in the decoder and make a point of documenting everything like this.  Fortunately you can just copy/adapt the comment from [here](https://github.com/trufflesuite/truffle/blob/develop/packages/codec/lib/types.ts#L735).

Reply:
Thanks @haltman-at!